src/Symfony/Component/CssSelector/Tests/XPath/TranslatorTest.php

```diff
@@ -116,7 +116,7 @@ public function testHtmlIds($css, array $elementsId)
         $this->assertCount(\count($elementsId), $elementsId);
         foreach ($elements as $element) {
             if (null !== $element->attributes()->id) {
-                $this->assertTrue(\in_array($element->attributes()->id, $elementsId));
+                $this->assertContains((string) $element->attributes()->id, $elementsId);
```

Comment:
This is not strictly related,  but it improves an error message shown when the test fails. Before the change, the message would say "expected true, got false". Now it will indicate what's the missing id.